notes-webapp/src/main/webapp/vue-app/notes-editor/components/TranslationsEditBar.vue

```diff
@@ -108,6 +108,7 @@
       <select
         id="translationBarFilterSelect"
         v-model="selectedLang"
+        aria-label="List of traductions"
```

Comment:
List of translations

Reply:
updated, thanks Patrice for the review :)